py/tests/integration/setup.yaml

```diff
@@ -107,7 +107,7 @@ tests: # test specific settings
     test_contract_create:
       nodes:
         node: dev1
-      blocks_to_mine: 115
+      blocks_to_mine: 5
```

Comment:
Could you please re-decrease timeout in  2 py/tests/integration/common.py wait_until_height from 600s to something more sensible e.g. previous 120s ?

https://github.com/aeternity/epoch/commit/4ef90d62a5d0e9affc79e29a1e83e6beaa06a552#diff-aaa0ff120e51b18e0a5d4dc49ea74dd8R189